.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
@alshaw This file is specific to your own machine and not the project, so it shouldn't be added to the project repository, but instead is something you would put in the global .gitignore on your machine: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

I put `.idea` in my global .gitignore so that everything in `.idea` is git-ignored everywhere.

Reply:
Oops, I thought I had already done that. Thanks Aaron.